examples/command/command.cpp

```diff
@@ -138,6 +139,9 @@ std::string transcribe(whisper_context * ctx, const whisper_params & params, con
     wparams.language         = params.language.c_str();
     wparams.n_threads        = params.n_threads;
 
+    // disable fallback - seems not useful for command recognition
+    wparams.temperature_inc  = 0.0f;
```

Comment:
This seems to make the command recognition example much more robust, although we want to enable it from time to time to make sure that multi-decoder usage is not broken.

Reply:
Seems the reason for this to improve is that the fallback logic depends on the average logprob of the generated sequences:

https://github.com/ggerganov/whisper.cpp/blob/b8f34d1ed786194d9787b1f1d086b89136e361f3/whisper.cpp#L5166-L5171

Therefore, I think both "best of" and "beam search" strategies would not work as expected.
We probably need to have some re-normalization of the logprobs after applying the grammar to make this compatible with these.

In any case, I think for now we should focus on greedy sampling without fallbacks and improve later if needed